tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -48,6 +48,9 @@ test.describe('SAML Login', () => {
 
     // Setups the SAML/SSO login with samltest.id Identity Provider
     test('Set up SAML', async ({ context, environment, baseUrl, page }) => {
+      // QA-579
```

Comment:
ah I guess I was a bit too excited there - the `browserName` is not yet available to check for... =>
```suggestion
    test('Set up SAML', async ({ browserName, context, environment, baseUrl, page }) => {
      // QA-579
```

Reply:
Good catch, fixed!